src/main/java/ru/skypro/homework/controller/RegisterController.java

```diff
@@ -0,0 +1,23 @@
+package ru.skypro.homework.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ru.skypro.homework.dto.RegisterDto;
+import ru.skypro.homework.service.impl.RegisterServiceImpl;
+
+@RestController
+@RequestMapping(name = "/register")
+@AllArgsConstructor
+public class RegisterController {
+
+    private final RegisterServiceImpl registerServiceImpl;
```

Comment:
```suggestion
    private final RegisterService registerServiceImpl;
```
инъекцию надо всегда делать по интерфейсу, а не реализации